plugins/inputs/opcua/opcua_client.go

```diff
@@ -13,11 +13,12 @@ import (
 	"github.com/influxdata/telegraf"
 	"github.com/influxdata/telegraf/config"
 	"github.com/influxdata/telegraf/plugins/inputs"
+	"github.com/influxdata/telegraf/selfstat"
 )
 
 // OpcUA type
 type OpcUA struct {
-	Name           string          `toml:"name"`
+	MetricName     string          `toml:"metric_name"`
```

Comment:
backwards support for name might be good here.

Reply:
I wasn't worried about backward compatibility when I started this PR because I thought this plugin was new enough and broken enough that there wouldn't be enough existing users to worry about.  Your and Sven's concerns made me change my mind so I've reverted this.